modules/tinymce/src/plugins/lists/main/ts/core/Selection.ts

```diff
@@ -48,13 +49,27 @@ const getSelectedListItems = (editor: Editor): Array<HTMLLIElement | HTMLElement
 const getSelectedDlItems = (editor: Editor): HTMLElement[] =>
   Arr.filter(getSelectedListItems(editor), NodeType.isDlItemNode);
 
-const getClosestListRootElm = (editor: Editor, elm: Node): HTMLElement => {
+const getClosestEditingHost = (editor: Editor, elm: Node): HTMLElement => {
   const parentTableCell = editor.dom.getParents<HTMLTableCellElement>(elm, 'TD,TH');
   return parentTableCell.length > 0 ? parentTableCell[0] : editor.getBody();
 };
 
+const isListHost = (schema: Schema, node: Node) => {
+  const nodeName = node.nodeName;
+  const listNames = [ 'UL', 'OL', 'DL' ];
```

Comment:
Should this be a module scope constant? If so we could also use it in the `getParentList` selector potentially.

Reply:
Yes, I'v been thinking we should have two constants one selector and one list types or something. Maybe I squeeze that cleanup into this PR.